invenio_accounts/config.py

```diff
@@ -31,3 +31,7 @@
 # ACCOUNTS_FORCE_BLUEPRINT_REGISTRATION can be use to override
 # ``register_blueprint`` option.
 ACCOUNTS_REGISTER_BLUEPRINT = None
+
+# Set the secure flag on the session cookie to ensure it is only transmitted
+# over a secure channel.
+SESSION_COOKIE_SECURE = True
```

Comment:
We have to make it work for using the development server as well. Like this, the dev server will be unusable as it's running http.....